app/src/ui/components/ThemeToggle.jsx

```diff
@@ -0,0 +1,38 @@
+import styles from "./ThemeToggle.module.css";
+import clsx from "clsx";
+import React, { useState } from "react";
+import { useTheme } from "../hooks/useTheme";
+
+const ThemeToggle = () => {
```

Comment:
This is a mixed component. We should split the logic into 2 peaces:

1. Switch.jsx - just a switch without theme logic
2. External component. In out case it is App.jsx where all of the theme chaning logic is gonna hapend

We should split the UI and business logic. Cause one day you will need to use this switcher for settings for example like in Iphone. But it contains logic for theme. Which is gonna pretty ugly.

Reply:
I got you)